napari/layers/tracks/_track_utils.py

```diff
@@ -253,17 +246,20 @@ def _validate_track_graph(
         self, graph: Dict[int, Union[int, List[int]]]
```

Comment:
Nitpick, and not your fault, but for a latter PR: `..._validate_..` tend to not return a new fixed item, but either a boolean or simply raise. I've seen many issues related to improper naming. Would suggest maybe renaming this to `..._normalize_...` ? Note that it _also_ validate later down, so just thinking at loud.

Is it still returning `Dict[int, Union[int, List[int]]]` or just `Dict[int, List[int]]` 


You seem to have spent quite a bit of time in this file, have you though of having `Graph = Dict[int, List[int]]` ? I guess if it's just `Dict[int, List[int]]` it might not be necessary.

Reply:
> You seem to have spent quite a bit of time in this file, have you though of having Graph = Dict[int, List[int]] ? I guess if it's just Dict[int, List[int]] it might not be necessary.

I like this idea in theory, but `mypy` raised a weird error when I tried to define `Graph = Dict[int, List[int]]` and then use it. Given there aren't many instances of this I think worth leaving for later to fix the `mypy` issue.

> Is it still returning Dict[int, Union[int, List[int]]] or just Dict[int, List[int]]

It's returing `Dict[int, List[int]]`, and that's how it's annotated.

> Would suggest maybe renaming this to ..._normalize_... ? 

👍 good idea, done.